index.html

```diff
@@ -12,16 +12,16 @@
   <div class="container">
   
     <header>
-      <img src="images.png/logo.png" alt="Logo for Adare Restaurant">
+      <img src="images.png/pho-crimson-ICON_512x512.png" alt="Logo for PHO KIM LONG">
```

Comment:
maybe we want to rename the logo to something generic like `logo.png`; also want to change casing on name
```suggestion
      <img src="images.png/logo.png" alt="Logo for Pho Kim Long">
```